### DISCUSSION TITLE
Import all existing issues in Github Projects

### DISCUSSION THREAD

**Discussion by iosifnicolae2** [community/community#40815] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi.

Would it be possible to import all existing issues in Github Projects?

Thank you!

**Comment by iosifnicolae2** [comment]
We can do the import manually. But would it be nice to have a column named Backlog where all the issues are imported automatically.

**Comment by lcameron05** [comment]
It would be useful to be able to map repositories to a project such that all existing and newly created issues are automatically added to the project. 

The "Auto-add to project" workflow currently in Beta nearly allows for this, but has the following limitations:
* can only open one such "auto-add" rule (and can therefore not serve multiple repositories)
* existing items must be added manually (would be create to be able to run the rule retrospectively)
